charts/k8tz/templates/controller.yaml

```diff
@@ -81,6 +92,27 @@ spec:
               scheme: HTTPS
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+        - name: {{ .Chart.Name }}-cert-watcher
```

Comment:
remind you that we decided that what you did in the beginning with enabling the cert watcher container only when cert-manager enabled (`.Values.webhook.certManager.enabled`) was better

Reply:
sure, will put the conditional checking back